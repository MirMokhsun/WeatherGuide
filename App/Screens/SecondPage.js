import React, { Component } from 'react'
import { TouchableOpacity, Text, ImageBackground } from 'react-native'
import { styles } from '../styles/MainScreenStyles'

export default class SecondPage extends Component {
    render() {
        return (
            <ImageBackground style={styles.container}
                resizeMode='cover'
                source={require('../assets/Images/Wallpaper.jpg')} >
                <Text style={styles.txt}>Second Page</Text>
                    <TouchableOpacity style={styles.WB} onPress={() => { this.props.navigation.navigate('Main') }}>
                    </TouchableOpacity>
            </ImageBackground>
        )
    }
}