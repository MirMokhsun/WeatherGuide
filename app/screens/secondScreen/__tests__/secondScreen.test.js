import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SecondPage from '../';

describe('SecondPage', () => {
    test('SecondPage snapshot', () => {
        const renderer = new ShallowRenderer();
        const snap3 = renderer.render(<SecondPage />);
        expect(snap3).toMatchSnapshot();
    });
});