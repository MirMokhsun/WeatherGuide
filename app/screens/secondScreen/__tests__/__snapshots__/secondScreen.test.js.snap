// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondPage SecondPage snapshot 1`] = `
<ImageBackground
  resizeMode="cover"
  source={
    Object {
      "testUri": "../../../app/assets/images/Background.jpg",
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(255,255,255,0.3)",
        "borderRadius": 3,
        "height": 75,
        "marginLeft": 5,
        "marginTop": 15,
        "width": 740,
      }
    }
  />
</ImageBackground>
`;
